src/app/cube/details/components/academic-level-card/academic-level-card.component.html

```diff
@@ -0,0 +1,18 @@
+<div class='academic-level-card'>
+  <div class='academic-level-card__icon-container'>
+      <i class="fab fa-angular fa-3x"></i>
+  </div>
+  <div class='academic-level-card__title-container'>
+    {{ category }}
+  </div>
+  <div class='academic-level-card__applied-levels-container'>
+    <div *ngFor='let level of academicLevels | keyvalue'>
```

Comment:
I think this should be transformed into a list for accessibility and usability.

Reply:
![image](https://user-images.githubusercontent.com/7431390/71186971-27dee780-224c-11ea-8e29-09bc718ac4d9.png)